commands/cli.go

```diff
@@ -186,14 +186,14 @@ func awsNuke(c *cli.Context) error {
 	}
 
 	// Log which resource types will be nuked
-	logging.Logger.Info("The following resource types will be nuked:")
+	logging.Logger.Info(" -> The following resource types will be nuked:")
 	if len(resourceTypes) > 0 {
 		for _, resourceType := range resourceTypes {
-			logging.Logger.Infof("- %s", resourceType)
+			fmt.Println("^ ->",resourceType)
 		}
 	} else {
 		for _, resourceType := range allResourceTypes {
-			logging.Logger.Infof("- %s", resourceType)
```

Comment:
This means that you can't suppress the logs using log leveling, which would be a regression. I think you can get the same view if you buffer the contents into a newline separated string and pass it in with `logging.Logger.Info`.